sycl/test-e2e/Matrix/joint_matrix_bfloat16_packedB_impl.hpp

```diff
@@ -13,7 +22,7 @@ void matrix_multiply(big_matrix<T1, M, N> &C, big_matrix<T2, M, K> &A,
      auto accA = bufA.get_access<access::mode::read_write>(cgh);
      auto accB = bufB.get_access<access::mode::read_write>(cgh);
 
-     cgh.parallel_for<class imatrix>(
+     cgh.parallel_for<kernel_name>(
          nd_range<2>({NDRangeM, NDRangeN * SG_SZ}, {1, 1 * SG_SZ}),
          [=](nd_item<2> spmd_item) [[intel::reqd_sub_group_size(SG_SZ)]]
```

Comment:
Remove the attribute when  get_sg_size is used.
Keep the attribute for SG_SIZE=32

Reply:
This is not the point of this PR